README.md

```diff
@@ -2,26 +2,29 @@
 ## OMS-core
 
 ## Description
-The core repository for the Online Management System (OMS), an open-source project of student association AEGEE-Europe.
+The core service repository for the [Online Management System (OMS)](https://github.com/AEGEE/oms-docker), an open-source project of student association [AEGEE-Europe](http://aegee.org/).
```

Comment:
Whoops. Fun fact: this is copied from the README in OMS-Docker, so you missed it there :D